docs/_includes/navbar.html

```diff
@@ -39,7 +39,7 @@
         <!-- The navigation links. -->
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
           <ul class="nav navbar-nav">
-            <li{% if page.url == '/index.html' %} class="active"{% endif %}><a href="{{ site.baseurl}}/index.html">Overview<span class="hidden-sm hidden-xs"> {{ site.version_short }}</span></a></li>
+            <li{% if page.url == '/' %} class="active"{% endif %}><a href="{{ site.baseurl}}/">Overview<span class="hidden-sm hidden-xs"> {{ site.version_short }}</span></a></li>
```

Comment:
We can for sure. But do we want? :) Why should it be better? I think "Overview" is good.